<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="myFantasyPlayerNameLabel">
      <property name="text">
       <string>FantasyPlayerName</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>fp status</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>489</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Balance</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="myBalanceLabel">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="myPreviousWeek">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mySeasonLabel">
        <property name="text">
         <string>Season label   week #</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="myNextWeek">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QStackedWidget" name="myStackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="myPreviousWeekView">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="PreviousWeekWidget" name="myPreviousWeekWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="myCurrentWeekView">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="CurrentWeekWidget" name="myCurrentWeekWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="myNexWeekView">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="NextWeekWidget" name="myNextWeekWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="myMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="myStatusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviousWeekWidget</class>
   <extends>QWidget</extends>
   <header>previousweekwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurrentWeekWidget</class>
   <extends>QWidget</extends>
   <header>currentweekwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NextWeekWidget</class>
   <extends>QWidget</extends>
   <header>nextweekwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
